packages/infra/lib/api-stack.ts

```diff
@@ -1494,6 +1504,37 @@ export class APIStack extends Stack {
     return fhirToMedicalRecordLambda;
   }
 
+  private setupFhirToCdaConverterLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    medicalDocumentsBucket: s3.Bucket;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, medicalDocumentsBucket, sentryDsn, alarmAction, envType } = ownProps;
+
+    const fhirToCdaConverterLambda = createLambda({
```

Comment:
We're using DLQ in case of failure? Let's have a plan to process errors, like if messages on the DLQ are idempotent, whether we just retry a few times and toss them away, or whether we need to act on each. And add it to the [Ops Cookbook](https://www.notion.so/metriport/Engineering-Ops-Cookbook-34413ce59e9b46c99e1f73422b928c2e?pvs=4)

Reply:
Discussed over Slack. Decided not to use DLQ for this, since this Lambda will be called synchronously